airflow/operators/dagrun_operator.py

```diff
@@ -93,5 +93,6 @@ def execute(self, context):
                         conf=json.dumps(dro.payload),
                         execution_date=self.execution_date,
                         replace_microseconds=False)
+            self.run_id = dro.run_id
```

Comment:
What is this change for?

Reply:
The dagrun operator needs to store the dagrun id  so that it can return it when it is called. For example, in the test_triggered_dagrun_sensor_dag.py, this function triggers the dag and returns the id:
```
def triggerchild():
        dr = trigger_dag(TEST_DAG_CHILD)
        return [dr.run_id]
```